Makefile

```diff
@@ -335,7 +335,7 @@ golint: get-ci-tools
 .PHONY: operator-lint
 operator-lint: $(LOCALBIN) gowork
 	GOBIN=$(LOCALBIN) go install github.com/gibizer/operator-lint@2ffa25b7f1c13fb2bdae5444a3dd1b5bbad5
-	go vet -vettool=$(LOCALBIN)/operator-lint ./... ./api/...
+	go vet -vettool=$(LOCALBIN)/operator-lint ./... ./api/..
```

Comment:
unsure about this change; where does the commit 2ffa25b7f1c13fb2bdae5444a3dd1b5bbad5 come from?
also, if we're passing the module path with ./... why do we need api separately? 

I ask because I get this error: 

```
make-operator-lint.......................................................Failed
- hook id: make-operator-lint
- exit code: 2

test -f go.work || go work init
go work use .
go work use ./api
go work sync
GOBIN=/Users/gouthamr/repos/manila-operator/bin go install github.com/gibizer/operator-lint@2ffa25b7f1c13fb2bdae5444a3dd1b5bbad5
go vet -vettool=/Users/gouthamr/repos/manila-operator/bin/operator-lint ./... ./api/...
pattern ./api/...: main module (github.com/openstack-k8s-operators/manila-operator) does not contain package github.com/openstack-k8s-operators/manila-operator/api/v1beta1
make: *** [operator-lint] Error 1
```